main.css

```diff
@@ -0,0 +1,83 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+#main {
+  background-color: silver;
+}
+
+#main h2 {
```

Comment:
Using ID as a selector is good but the best practice is to use another alternative like classes and tags,etc